metrics/metrics.go

```diff
@@ -175,6 +178,17 @@ func NewCSIMetricsManagerForSidecar(driverName string) CSIMetricsManager {
 	return NewCSIMetricsManagerWithOptions(driverName)
 }
 
+// NewCSIMetricsManagerForSidecarWithMigrationStatus creates and registers metrics
+// for CSI Sidecars and returns an object that can be used to trigger the metrics.
+// It uses "csi_sidecar" as subsystem. Meanwhile, it adds the migrated label to
+// the metrics to indicate if the operation is from a csi migration
+//
+// driverName - Name of the CSI driver against which this operation was executed.
+//              If unknown, leave empty, and use SetDriverName method to update later.
+func NewCSIMetricsManagerForSidecarWithMigrationStatus(driverName string) CSIMetricsManager {
```

Comment:
I don't think we need this helper function. Just document how to enable this feature and then let sidecars call `NewCSIMetricsManagerWithOptions(driverName, WithLabelNames(LabelMigrated))`.

Reply:
Sure thing. I will remove it.